spanner/api/Spanner/GetBackups.cs

```diff
@@ -0,0 +1,107 @@
+// Copyright 2020 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START spanner_get_backups]
+
+using Google.Api.Gax;
+using Google.Cloud.Spanner.Admin.Database.V1;
+using Google.Cloud.Spanner.Common.V1;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace GoogleCloudSamples.Spanner
+{
+    public class GetBackups
+    {
+        public static object SpannerGetBackups(
+            string projectId, string instanceId, string databaseId, string backupId)
+        {
+            // Create the DatabaseAdminClient instance.
+            DatabaseAdminClient databaseAdminClient = DatabaseAdminClient.Create();
+
+            InstanceName parentAsInstanceName = InstanceName.FromProjectInstance(projectId, instanceId);
+
+            Action<List<Backup>> printBackups = backups =>
```

Comment:
Consider defining a local method instead, at the end of this one:

void PrintBackups(List<Backup> backups) {...}

Reply:
Done!